app/services/solidus_bolt/transaction/capture.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module Transaction
+    class Capture < SolidusBolt::Base
+      def initialize(transaction_reference:, amount:, currency:)
+        @transaction_reference = transaction_reference
+        @amount = amount
+        @currency = currency
+        super
+      end
+
+      def call
+        capture
+      end
+
+      private
+
+      def capture
+        options = build_options
+        HTTParty.post(
+          "#{api_base_url}/#{api_version}/merchant/transactions/capture",
+          options
+        )
+      end
+
+      def build_options
+        options = {
+          query: {
+            transaction_reference: @transaction_reference,
+            amount: @amount,
+            currency: @currency
```

Comment:
wouldn't it be better to use attr_readers?

Reply:
using attr_readers now